okgo/check/check_test.go

```diff
@@ -112,3 +116,18 @@ func TestRun_ErrorsButFilteredOut(t *testing.T) {
 	err := Run(projectParam, checkersToRun, []string{"p1"}, "dir", nil, 2, os.Stdout)
 	assert.NoError(t, err)
 }
+
+func TestRun_ErrorsOnTypeCheck(t *testing.T) {
+	projectParam := okgo.ProjectParam{
+		Checks: map[okgo.CheckerType]okgo.CheckerParam{
+			"error_on_type_check": {
+				Checker: &inMemoryChecker{checkerType: "error_on_type_check"},
+			},
+		},
+	}
+	checkersToRun := []okgo.CheckerType{
+		"error_on_type_check",
+	}
+	err := Run(projectParam, checkersToRun, nil, "dir", nil, 2, os.Stdout)
```

Comment:
Is it useful to pass `os.Stdout` here to see output, or should we just do `io.Discard` or similar?

Reply:
Fixed